mino/minoch/mod.go

```diff
@@ -12,15 +12,20 @@ import (
 	"go.dedis.ch/dela/serde/json"
 )
 
+// Filter is a function called for any request to an RPC which will drop it if
+// it returns false.
+type Filter func(mino.Request) bool
```

Comment:
isn't it a bit expensive to add all this mechanism just for the sake of testing ?

Reply:
Minoch has been implemented for the sake of testing, so I don't think so. It's also very cheap as the default behaviour of being empty does add anything to the logic.